postgresql-persistence/src/main/java/com/netflix/conductor/aurora/AuroraIndexDAO.java

```diff
@@ -0,0 +1,219 @@
+package com.netflix.conductor.aurora;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.netflix.conductor.common.metadata.events.EventExecution;
+import com.netflix.conductor.common.metadata.tasks.Task;
+import com.netflix.conductor.common.metadata.tasks.TaskExecLog;
+import com.netflix.conductor.common.run.SearchResult;
+import com.netflix.conductor.common.run.Workflow;
+import com.netflix.conductor.core.events.queue.Message;
+import com.netflix.conductor.dao.IndexDAO;
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.joda.time.DateTime;
+import org.joda.time.format.ISODateTimeFormat;
+
+import javax.inject.Inject;
+import javax.sql.DataSource;
+import java.sql.Timestamp;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.LinkedList;
+import java.util.List;
+
+public class AuroraIndexDAO extends AuroraBaseDAO implements IndexDAO {
+
+	@Inject
+	public AuroraIndexDAO(DataSource dataSource, ObjectMapper mapper) {
+		super(dataSource, mapper);
+	}
+
+	@Override
+	public void index(Workflow workflow) {
+		// Do nothing here
```

Comment:
Why don't we need to do any work in these methods?

Reply:
The index dao logic and execution dao logic merged into execution dao.